values.yaml

```diff
@@ -325,43 +325,46 @@ server:
 externalServers:
   # If true, the Helm chart will be configured to talk to the external servers.
   # If setting this to true, you must also set server.enabled to false.
-  # Note that if you are setting client.join property, https.address property is not required.
   enabled: false
 
-  # HTTPS configuration for external servers.
-  # Note: HTTP connections to the servers are
-  # not supported.
-  https:
-    # IP, DNS name, or Cloud auto-join string pointing to the external Consul servers.
-    # Port must be provided separately with the externalServers.https.port property.
-    # Note that if you’re providing the cloud auto-join string and multiple addresses
-    # can be returned, only the first address will be used.
-    # This value is required only if you would like to use
-    # a different server address from the one specified
-    # in the client.join property.
-    address: null
-
-    # The HTTPS port of the server.
-    port: 443
+  # An array of external Consul hosts.
+  # Valid values include IPs, DNS names, or Cloud auto-join string.
+  # Ports must be provided separately below.
+  # These hosts are used for clients to join the server cluster,
+  # as well as for HTTP/HTTPS connections from the components in this Helm chart.
+  hosts: []
+
+  # The Serf LAN port that the clients will use to join the server cluster.
+  serfLANPort: 8301
+
+  # The HTTPS port of the Consul servers.
+  httpsPort: 443
 
-    # tlsServerName is the server name to use as the SNI
-    # host header when connecting with HTTPS.
-    # This property is useful in case ‘externalServers.https.address’
-    # is not or can not be included in the server certificate’s SANs.
-    tlsServerName: null
+  # tlsServerName is the server name to use as the SNI
+  # host header when connecting with HTTPS.
+  tlsServerName: null
 
-    # If true, the Helm chart will ignore the CA set in
-    # global.tls.caCert and will rely on the container's
-    # system CAs for TLS verification when talking to Consul servers.
-    # Otherwise, the chart will use global.tls.caCert.
-    useSystemRoots: false
+  # If true, the Helm chart will ignore the CA set in
+  # global.tls.caCert and will rely on the container's
+  # system CAs for TLS verification when talking to Consul servers.
+  # Otherwise, the chart will use global.tls.caCert.
+  useSystemRoots: false
+
+  # If you are setting global.acls.manageSystemACLs and connectInject.enabled to true,
+  # set k8sAuthMethodHost to the address of the Kubernetes API server.
+  # This address must to be reachable from the Consul servers.
+  # Please see https://www.consul.io/docs/acl/auth-methods/kubernetes.html.
+  # Requires consul-k8s >= 0.14.0.
+  k8sAuthMethodHost: null
```

Comment:
Can you add an example, e.g. `https://7D96FAF246DABD092E8B0E04B73D967C.sk1.us-east-1.eks.amazonaws.com`

Reply:
I'll provide a command to retrieve from your kube config.